README.md.orig

```diff
@@ -0,0 +1,127 @@
+<<<<<<< HEAD
```

Comment:
This file shouldn't be committed. It looks like it is part of the merge conflict.